src/onyx/state/bookkeeper.clj

```diff
@@ -43,6 +43,7 @@
                                                 (.setZkServers (:zookeeper/address env-config))
                                                 (.setZkLedgersRootPath ledgers-root-path)
                                                 (.setBookiePort port)
+                                                ;(.setAutoRecoveryDaemonEnabled true)
```

Comment:
This was a WIP that shouldn't have been committed as part of this PR. Leaving it since I'm going to enable it shortly.